### DISCUSSION TITLE
install the CPU version 3.0.0b1 of deepmd

### DISCUSSION THREAD

**Discussion by poyaii** [deepmodeling/deepmd-kit#3999] [Q&A]
Hello, thank you for the development of deepmd-kit v3. My goal is to install the CPU version 3.0.0b1 on Ubuntu. I used this command from the Anaconda websitehttps://anaconda.org/conda-forge/deepmd-kit/files : conda install conda-forge/label/deepmd-kit_rc::deepmd-kit, but it gives an error. **How can I install the CPU version 3.0.0b1 using the conda command?** I would appreciate your guidance.  

![image](https://github.com/user-attachments/assets/d596d98b-4684-417f-944d-8edce9073677)

**Comment by njzjz** [comment]
Try `conda install conda-forge/label/deepmd-kit_rc::deepmd-kit -c conda-forge` as its dependencies are in the `conda-forge` channel.
